drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -350,7 +350,8 @@ def full_name
   end
 
   def build_client_headshot_file(image_blob_id, current_user)
-    current_image_blob = ActiveStorage::Blob.find_by(id: image_blob_id)
+    # TODO(#7054) - accept either unsigned or signed blob ID for now, for backwards compatibility
```

Comment:
What's the lifespan of these?  Could we remove this one week after this is deployed?  
Are we worried someone would have an unsigned url in a bookmark or email?

Reply:
Yes, I think a week after deploy sounds about right. These aren't URLs so I'm not worried about bookmarks or saved links. We don't support "save and finish later" for assessments that include file uploads. I guess it would cause an error if someone uploaded a file in a form and then left the tab open for weeks without submitting, but that seems very unlikely.

Reply:
+1 to removing this in the subsequent release